svg.js.d.ts

```diff
@@ -1243,6 +1255,7 @@ declare module "@svgdotjs/svg.js" {
         stop(jumpToEnd: boolean, clearQueue: boolean): Animation;
         stop(offset?: NumberAlias | string, color?: NumberAlias, opacity?: NumberAlias): Stop;
         stop(val: { offset?: NumberAlias | string, color?: NumberAlias, opacity?: NumberAlias }): Stop;
+        stroke(): any;
         stroke(color: string): this;
         stroke(stroke: StrokeData): this;
         tbox(): Box;

```

Comment:
This one can go. It doesnt exist anymore

Reply:
I can take it out, but it looks to me like it is there. Or do i misunderstand?

https://github.com/svgdotjs/svg.js/blob/9e75004466f251d765f72fd80fad846e71fae107/src/modules/optional/sugar.js#L24

Reply:
You have a reference to sugar.js in your answer - is this wanted? Because I dont get it :D.
We dont have a tbox. We have rbox and bbox. tbox was deprectated long time ago and replaced by rbox

Reply:
i'm sorry, i thought you were referring to the green line (`stroke(): any`). Got it :)

Reply:
Ah okay :D. Btw, can we meet at gitter for a few minutes? I am wtriting typings for the filter plugin and have one weired error popping up in the d.ts file